src/test/scala/com/raphtory/lotrtest/LotrTest.scala

```diff
@@ -146,6 +147,13 @@ class LotrTest extends BaseRaphtoryAlgoTest[String] {
     assert(true)
   }
 
+  test("Prisoners Dilemma Test") {
+    assert(
+      algorithmTest(PrisonersDilemma(proportionCoop = 0.9f, benefit = 1.05f, cost = 0.05f,seed = 42),outputFormat,1, 32674, 10000, List(500, 1000, 10000))
```

Comment:
Algo and doc strings look great! We are having some issues at the moment where the seeded algorithms will return different results with different numbers of partitions, but this is a wider problem and will do the trick for the time being 😄

Reply:
Ahhh ok yeah that makes a lot of sense! 😃